bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
+            - GOODS_AND_SERVICES_TAX
+            - TIP_OR_GRATUITY
+            - FOOD_AND_DRINKS
+            - WIFI
+            - BUS_FARE
+      excluded:
+        type: string
+        description: 'Optional text describing what is excluded when you book this product. May contain HTML markup.'
+      requirements:
+        type: string
+        description: 'The text describing what you need to bring for this activity. May contain HTML markup.'
+      knowBeforeYouGoItems:
         type: array
+        items:
+          type: string
+          enum:
+            - STROLLER_OR_PRAM_ACCESSIBLE
+            - WHEELCHAIR_ACCESSIBLE
+            - LIMITED_MOBILITY_ACCESSIBLE
+            - LIMITED_SIGHT_ACCESSIBLE
+            - ANIMALS_OR_PETS_ALLOWED
+            - PUBLIC_TRANSPORTATION_NEARBY
+            - INFANT_SEATS_AVAILABLE
+            - INFANTS_MUST_SIT_ON_LAPS
+            - PASSPORT_REQUIRED
+            - DRESS_CODE
       attention:
         type: string
         description: 'Text describing any points that need special attention for the passengers. May contain HTML.'
-      baseLanguage:
-        type: string
-        description: 'The language that the product description is written in. There may also be translations available.'
-      bookableExtras:
+      locationCode:
+        $ref: '#/definitions/LocationCodeDto'
+        description: 'The city that represents the main location for this activity. Specified as a location from the United Nations Code for Trade and Transport Locations (UN/LOCODE).'
+      googlePlace:
+        type: object
+        properties:
+          country:
+            type: string
+          countryCode:
+            type: string
+          city:
+            type: string
+          cityCode:
+            type: string
+          geoLocationCenter:
+            type: object
+            description: 'Represents a point in geographical coordinates: latitude and longitude.'
+            properties:
+              lat:
+                type: number
+                format: double
+              lng:
+                type: number
+                format: double
+      resourceSlots:
         type: array
-        description: 'The list of extras available for this product. Note that not all of these extras may be available for all bookable items. For example, different rates may offer different extras.'
         items:
-          $ref: '#/definitions/BookableExtraDto'
-      bookingCutoffDays:
+          type: object
+          properties:
+            id:
+              type: integer
+              format: int64
+            count:
+              type: integer
+              format: int32
+        description: 'Not working.'
+      bookingCutoffMinutes:
         type: integer
         format: int32
-        description: 'The number of days before the activity when we stop accepting bookings.'
+        description: 'The number of minutes before the activity when we stop accepting bookings.'
```

Comment:
Worth mentioning that all 4 fields play together, e.g. if hours=1 and minutes=1, then the overall amount is 1h 1m. Copy that for all other fields.

Reply:
done